src/PKSim.Core/Services/PKSimParameterStartValuesCreator.cs

```diff
@@ -75,40 +58,6 @@ private bool isChangedSimulationParameter(IParameter parameter)
                 && parameter.ValueDiffersFromDefault();
       }
 
-      private void updateMoleculeParametersValues(IndividualMolecule molecule, Individual individual, Simulation simulation)
-      {
-         var allMoleculeParameters = individual.AllMoleculeParametersFor(molecule);
-         var modelConfiguration = simulation.ModelConfiguration;
-         allMoleculeParameters.Each(p => setParameter(p, modelConfiguration.ModelName == CoreConstants.Model.FOUR_COMP));
-      }
-
-      private void setParameter(IParameter parameter, bool isSmallMolecule)
-      {
-         //We do not generate start values for endosome parameters when dealing with a small molecule model
-         if (parameter.HasAncestorNamed(CoreConstants.Compartment.ENDOSOME) && isSmallMolecule)
-            return;
-
-         //We do not generate value for this parameter by default. Exit
-         var parameterStartValue = getStartValueFor(parameter);
-         if (parameterStartValue == null)
-            return;
-
-         if (parameter.Formula.IsExplicit())
-         {
-            var formula = _formulaFactory.RateFor(EXPRESSION_PARAMETERS, parameter.Formula.Name, _defaultStartValues.FormulaCache);
-            parameterStartValue.Formula = formula;
-            //There is a formula, make sure we use it. We set this flag to false to ensure that the formula will not be replaced with a constant formula
-            parameterStartValue.OverrideFormulaWithValue = false;
```

Comment:
OverrideFormulaWithValue  I am not sure that this flag should be kept now. This was because we were creating values on the fly for everything and wanted to keep the formula even for constant value. But in this case...we won't do that anymore
@Yuri05 I'd vote to remove this flag. Thought?